submissions/dragalur/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" >
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" >
+		<title>Document</title>
+		<link rel="stylesheet" href="style.css" >
+	</head>
+	<body>
+		<div class="wrapper">
+			<div class="header">
+				<nav class="menu">
+					<ul class="menu-list">
+						<li>
+							<a class="menu-link"  href="">Mail</a>
+						</li>
+						<li>
+							<a class="menu-link" href="">Image</a>
+						</li>
+						<li class="focuses">
+							<label for="navbar-button" class="navbar-button">
```

Comment:
Your popup not accessible from keyboard. [Watch video](https://share.vidyard.com/watch/HbZ4qv7aYGvUC8GfCuC5DP?) Please fix it

Reply:
Not fixed yet.